src/components/App/App.js

```diff
@@ -41,14 +41,14 @@ function App() {
       <Search/>
       <main className="App">
         <Routes>
-          <Route path="/login" element={<Login currentUser={loggedInUser} onLogin={handleLogin} />} />
-          {/* <Route path="/" /> */}
+          <Route path="/login" element={<Login onLogin={handleLogin} />} />
+          <Route path="/" />
```

Comment:
looks like this change may have broken our login - tests are failing

Reply:
actually i'm not sure what's breaking it

Reply:
I say ignore this for now since we'll have to refactor quite a few things for the login. I also changed things to remove a mock user as part of my branch. Probably a little too far since it was not part of my feature but I got carried away when removing the alerts.

Reply:
login is still working on main tho

Reply:
i'd prefer not to leave that broken, even if we do have a refactor planned.... what is the planned refactor?